src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -157,5 +165,68 @@ protected void AddPagingPerformanceHint(PagingOperationType operation, string ac
             if(ShouldAddPagingPerformanceHint(numberOfResults))
                 Database.NotificationCenter.Paging.Add(operation, action, details, numberOfResults, pageSize, duration, totalDocumentsSizeInBytes);
         }
+
+        protected DynamicJsonValue GetCustomConfigurationAuditJson(string name, BlittableJsonReaderObject configuration)
+        {
+            switch (name)
+            {
+                case "update-periodic-backup":
+                    return JsonDeserializationServer.BackupConfiguration(configuration).ToAuditJson();
+                case "etl-add":
```

Comment:
this is the same as `"put-connection-string"`, no? can we reuse the code in both places (make internal method)

Reply:
Done